diff -ur clucene-core-2.3.3.4.orig/src/shared/CLucene/clucene-config.h.cmake clucene-core-2.3.3.4/src/shared/CLucene/clucene-config.h.cmake
--- clucene-core-2.3.3.4.orig/src/shared/CLucene/clucene-config.h.cmake	2011-03-17 01:21:07.000000000 +0100
+++ clucene-core-2.3.3.4/src/shared/CLucene/clucene-config.h.cmake	2023-12-08 22:34:57.498531196 +0100
@@ -101,7 +101,7 @@
 
 /* Define to 1 if you can safely include both <sys/time.h> and <time.h>. */
 //not actually used for anything...
-//#cmakedefine _CL_TIME_WITH_SYS_TIME  1 
+#cmakedefine _CL_TIME_WITH_SYS_TIME  1 
 
 /* Define that we will be using -fvisibility=hidden, and 
  * make public classes visible using __attribute__ ((visibility("default")))
